UI changes and misc improvements for dataprep (#4800)
SoftwareAG/c8y-docs: content/data-preparation/getting-started-dataprep.md

### Step 1: Create your first rule {#create-rule}

1. Open the Data Preparation application. The main page shows your list of rules.
2. Click **Create rule** to open the rule creation wizard.

The wizard has three steps:

> 1. **Source** — configure which messages will be handled by this data preparation rule, by specifying the transport and at least one filter:

Reviewer: ```suggestion
1. **Source** — configure which messages will be handled by this Data Preparation rule by specifying the transport and at least one filter:
```

   - **Source transport**: select the transport from which messages are received. Currently this is always MQTT.

>    - **Topic filter**: filter based on the MQTT topic the device messages are received from. Use `*` as a wildcard to match zero or more characters, for example  `/sensors/factories/France/*/temperature/*` captures all temperature sensor data from that region. Note that transport-specific wildcards such as `+` and `#` are not supported here. 

Reviewer: ```suggestion
   - **Topic filter**: filter based on the MQTT topic the device messages are received from. Use `*` as a wildcard to match zero or more characters, for example,  `/sensors/factories/France/*/temperature/*` captures all temperature sensor data from that region. Note that transport-specific wildcards such as `+` and `#` are not supported here. 
```

>    - **Client ID filter**: if you do not know the topic, filter by the client ID of your device or gateway instead. Use `*` as a wildcard if needed, for example `serialnumber-1aaaf2ac-*`, `device-gateway-1234*`.

Reviewer: ```suggestion
   - **Client ID filter**: if you do not know the topic, filter by the client ID of your device or gateway instead. Use `*` as a wildcard if needed, for example, `serialnumber-1aaaf2ac-*`, `device-gateway-1234*`.
```

>    If your device is already connected to Cumulocity you can now capture a device message to test with using the source data you entered. Alternatively you can skip the live capture and enter your device messages for testing manually after creating the rule. 

Reviewer: ```suggestion
   If your device is already connected to {{< product-c8y-iot >}}, you can now capture a device message to test with using the source data you entered. Alternatively, you can skip the live capture and enter your device messages for testing manually after creating the rule. 
```
